### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amynicol1985**

### CODE REVIEW

**Comment by amynicol1985** [comment]
@jasonblais - I am working on updating the MLT sections of the handbook because they are considerably out of date. 
This is a 1% draft that I will get updated when the planning cycle is complete.

**Comment by jasonblais** [comment]
@amynicol1985 Looks like an excellent iteration, is this fine to move to editor review & pushing live?

File: operations/operations/mlt-cadence/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This section outlines:
 
 * Ian Tien: Co-Founder and CEO
 * Corey Hulen: Co-Founder and CTO
-* Tim Quock: VP, Finance
+* Kendra Niedziejko: CFO
 * David Reardon: VP Worldwide Sales
 * Chris Overton: VP, Engineering
 * Nirosha Ruwan: VP, Legal
```
Comment (cwarnermm) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* Nirosha Ruwan: VP, Legal
```

File: operations/operations/mlt-cadence/README.md [hunk 2]
```diff
@@ -19,9 +19,8 @@ This section outlines:
 
 * Ian Tien: Co-Founder and CEO
 * Corey Hulen: Co-Founder and CTO
-* Tim Quock: VP, Finance
+* Kendra Niedziejko: CFO
 * David Reardon: VP Worldwide Sales
-* Chris Overton: VP, Engineering
 * Nirosha Ruwan: VP, Legal
```
Comment (cwarnermm) [inline operations/operations/mlt-cadence/README.md:24]:
```suggestion
* Nirosha Ruwan: VP, Legal
* Chen-i Lim: VP, Product
```

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**